sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedStartFromBeginningImpl.java

```diff
@@ -16,10 +16,12 @@ public ChangeFeedStartFromBeginningImpl() {
     public void populatePropertyBag() {
         super.populatePropertyBag();
 
-        setProperty(
-            this,
-            Constants.Properties.CHANGE_FEED_START_FROM_TYPE,
-            ChangeFeedStartFromTypes.BEGINNING);
+        synchronized(this) {
```

Comment:
Does base class method call need to be "synchronized"/single threaded as well?

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.